l10n/bundle.l10n.de.json

```diff
@@ -1,9 +1,11 @@
 {
+  "Active file": "Active file",
```

Comment:
hey @scnjdl . Can you revert the changes in this file for now? We'll create a new PR at some point in the future to cover these new strings.